editor/assets/stylesheets/_variables.scss

```diff
@@ -19,6 +19,7 @@ $light-gray-300: #edeff0;
 $light-gray-200: #f3f4f5;
 $light-gray-100: #f8f9f9;
 $white: #fff;
+$red: #dc3232;
```

Comment:
Yes, let's use the $alert-red color, which I'm taking from Hugo's proposed new colors. We are tracking whether to use those, and have them influence the rest of the admin (which I'm a proponent for), or use the others. In the case of the latter it's extra important that it's a variable so it's easy to address if need be. Here are Hugo's colors:

https://codepen.io/hugobaeta/full/RNOzoV/

See also #573